.eslintrc.js

```diff
@@ -22,6 +22,11 @@ module.exports = {
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'max-len': ['off'],
     'import/prefer-default-export': ['off'],
+    camelcase: 'off',
+    'vuejs-accessibility/no-autofocus': 'off',
+    'vue/multi-word-component-names': ['error', {
+      ignores: ['Sidebar'],
+    }],
```

Comment:
Why add these rules? (And, what do they do?)

Reply:
In order to allow for the optional chaining operator, I had to update our babel and linting dependency versions. Some of those presets had been updated to include new rules by default, some of which aren't useful to us. So this change just reverts that.